src/projetos/HomeProjetosListPage.tsx

```
@@ -25,8 +25,9 @@ import FormPng from '@/assets/Form.png'
 import ProjetoService from '@/projetos/ProjetoService'
 import SugestaoService from '@/projetos/SugestaoService'
 import { AuthContext } from '@/context/authContext.tsx'
-import { IoArrowBackCircleOutline, IoArrowForwardCircleOutline } from "react-icons/io5";
+import { IoArrowBackCircleOutline, IoArrowForwardCircleOutline } from "react-icons/io5"
```

ESLint

[error] Replace `·IoArrowBackCircleOutline,·IoArrowForwardCircleOutline·}·from·"react-icons/io5"` with `⏎··IoArrowBackCircleOutline,⏎··IoArrowForwardCircleOutline,⏎}·from·'react-icons/io5'`

(prettier/prettier)